specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/networkVerifier.json

```diff
@@ -0,0 +1,1247 @@
+          "type": "string",
+          "description": "Destination resource id to verify the reachability path of."
+        },
+        "ipTraffic": {
+          "$ref": "#/definitions/IPTraffic",
+          "description": "IP traffic information."
+        }
+      }
+    },
+    "ReachabilityAnalysisRunListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ReachabilityAnalysisRun"
+          },
+          "description": "Gets a page of Reachability Analysis Runs."
+        },
+        "nextLink": {
+          "description": "Gets the URL to get the next page of results.",
+          "type": "string"
+        }
+      },
+      "description": "A list of Reachability Analysis Run"
+    },
+    "ReachabilityAnalysisRun": {
+      "required": [
+        "properties"
+      ],
+      "description": "Configuration information for analysis run.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ReachabilityAnalysisRunProperties"
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system metadata related to this resource.",
+          "$ref": "./network.json#/definitions/SystemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/CommonProxyResource"
+        }
+      ]
+    },
+    "ReachabilityAnalysisRunProperties": {
+      "required": [
+        "intentId"
+      ],
+      "description": "Represents the Reachability Analysis Run properties.",
+      "type": "object",
+      "properties": {
+        "description": {
+          "type": "string"
+        },
+        "intentId": {
+          "type": "string",
+          "description": "Id of the intent resource to run analysis on."
+        },
+        "intentContent": {
+          "readOnly": true,
+          "$ref": "#/definitions/IntentContent",
+          "description": "Intent information."
+        },
+        "analysisResult": {
+          "type": "string",
+          "readOnly": true
+        },
+        "errorMessage": {
+          "type": "string",
+          "readOnly": true
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        }
+      }
+    },
+    "VerifierWorkspaceListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VerifierWorkspace"
+          },
+          "description": "Gets a page of Verifier Workspaces."
+        },
+        "nextLink": {
+          "description": "Gets the URL to get the next page of results.",
+          "type": "string"
+        }
+      },
+      "description": "A list of Verifier Workspace"
+    },
+    "VerifierWorkspace": {
+      "description": "Instance of Verifier Workspace.",
+      "type": "object",
+      "properties": {
```

Comment:
>properties [](http://example.com/codeflow?start=7&length=10)

compared with the private version this one is missing the 

"location": {
                    "type": "string",
                    "x-ms-mutability": [
                        "create",
                        "read"
                    ]
                },

Reply:
removed those as they were duplicates and causing error in sdk generation for another team, already approved and merged here https://github.com/Azure/azure-rest-api-specs/pull/30327